### DISCUSSION TITLE
[Discussion] Mandatory & Optional Resource Burns - Building Maintenance

### DISCUSSION THREAD

**Discussion by TheDcol** [sunflower-land/sunflower-land#2977] [Ideas]
### **Introduction**

_This is a discussion on ideas and potential ways the game could introduce mandatory & optional resource burns in the game.
As it stands now, resources allow us to perform the following tasks:_

- _Make food_
- _Expand land_
- _Sell on market/shop_
- _Use for crafting new buildings/items/sfts_

_There are other ways a player can burn resources such as Treasure Island and Seasonal Events, although, these are optional burns the player can choose to avoid. For more on this optional vs mandatory burns please read prior discussion [[Game Economy Discussion] Resource Burning](https://github.com/sunflower-land/sunflower-land/discussions/2379)_

### **Building Maintenance**

I would like to have a discussion around introducing maintenance costs for certain buildings. If you are asking "why', please read the discussion linked above ^

How would it work? Let's just focus on the buildings that produce food for the moment:

![image](https://github.com/sunflower-land/sunflower-land/assets/108110065/046b92d3-2bc4-49f0-b88a-8c6c621fcd12)

**_DO NOTE: Just including SOME of the buildings and the resources used could differ, purely as example. I am after feedback around the MAIN IDEA of this mechanic, please do not comment "Deli should not need Iron grrr" :D_**

- Firepit would require no resources for it to stay open, although, players can choose to add wood to boost the firepit's production speed. We can determine exact values later but one example could be 'Add 10 wood to Firepit to cook 10-20% quicker for next 6-12 hours'.

- Kitchen would require x wood to stay open. Inputting wood in the kitchen would allow it to stay open for z days. A player may choose to also input stones to provide a boost in cooking speeds.

The resources listed in the table above are only examples, we could play around with different resources as well or even changing them up through the year. 

### **PROS:**

1. Resources used for core gameplay mechanics
2. Added depth to buildings and strategy

### **CONS**

1. Will need to evaluate a new player's journey; how this will affect their progression **(This is really important)**
2. Adding mandatory burns may put players off. Although some players are begging for more burns in the game, other players may simply be against it and would prefer to have more SFTs/new items to craft (Although, as my prior discussion outlinedd this has not addressed the overinflating resource economy)
3. May be too complex/stressful

### **Summary**

The main point of the discussion is whether this or other mandatory burns would be a good idea for the game; Does it provide extra strategies for the players? Does it add the right amount of complexity? Does the game need such a mechanic and more?

This idea could flow onto other buildings in the game. I would also like to explore other mandatory burn mechanics that would require players to use their resources on their farm and core gameplay to maintain and grow their farm. Other ideas I could explore are as follows:

- Pest control (Rats, locusts)
- Weeds (Weeds growing on plots if soil not looked after, weeds would cover a plot preventing a player to plant)
- Water (New water mechanic. Crops requiring water to grow, water would act as a new resource. Would tie in with the Weather & Seasons mechanic below)
- Weather & Seasons (Changing weather events and seasons. Certain crops may grow quicker/slower and yield more/less in certain weather/seasonal situations. Would tie in with the water mechanic above)

Players are asking for more burns and sinks for their resources, these are some of the ideas I have. As stated in the other discussion, intricate burn mechanics within the gameplay are better IMO over 'new item' burns to achieve a sustainable game economy.

Appreciate any constructive feedback or thoughts. Thank you <3

**Comment by eliasSFL** [comment]
I like the idea of maintenance using resources, maybe a durability aspect might be better rather than paying wood for a certain amount of days? If it's pay an amount of wood for x days to open players can pay like 1000 wood once and never have to pay until they run out, this doesn't make it that constant of a resource sink

**Reply by TheDcol** [reply]
>  If it's pay an amount of wood for x days to open players can pay like 1000 wood once and never have to pay until they run out, this doesn't make it that constant of a resource sink

I am picturing a player needing to pay a small amount per week, not in bulk.

Maybe an option to pay in bulk with less return. Eg:

- pay 10 wood for a week
- pay 100 wood for 7 weeks

It's not intended to solve the resource inflation issue on its own, there would be other mechanics to complement it

**Comment by adamhannigan** [comment]
Nice writeup @TheDcol 👏 

I do see buildings as a good starting point as a maintenance feature, especially since their isn't any growth/progression aspect once a player has unlocked all buildings.

As you mentioned, it could become stressful or annoying needing to repair too often.

What do you think the sweet spot is for requiring maintenance on buildings that wouldn't annoy players? It would be good to figure this out and the amount of resources that would be burnt to get a feel for what overall burn impact it would have on the game.

If this feature was to be added to the game it would be important to frame to gamify it so the experience is enjoyable: lore, progression, achievements (e.g. Master Repairer).

**Reply by TheDcol** [reply]
> What do you think the sweet spot is for requiring maintenance on buildings that wouldn't annoy players?

A small amount, as little as 5-10 wood for the week. It would be foolish to think that all the 'burn issues' can be solved with 1 mechanic, i believe small requirements spread across aspects of the game would be better than requiring 1 mechanic to cop the brunt of all burn solutions.

As far as making it enjoyable:
- Change the visual of building depending on whether it's closed or open
- Expand the lore, workers now demand farmers to pay for its maintenance
- Actionable boosts, a 1-time use item is dropped to a player if they manage to keep the building open for 4-6 weeks in a row (item could insta-cook a food in the building)

**Reply by rodolfoddias** [reply]
We could also add a lore where the crows from this season kind of stayed around Sunflower Isles and now they're attacking the farms crops. If you have enough scarecrows placed in your land, you are able to scare them. However, some of the crows started to attack the scarecrows and the players must repair them to keep the crows away and their crops safe.

**Comment by rodolfoddias** [comment]
I believe it's a must, specially during a bear market, to keep implementing new resource burns to control emissions. Specially because we know we have a lot of players hoarding resources until it's close to next halving date. 

I would like to propose the introduction of tiered tools as an Optional Resource Burn, that would result in a constant and daily burn of the entire mineral chain.

--

### [CONTEXT]

Until today, TI is the only active daily burning mechanism for resources like wood, stones, iron and gold. Since the Emerald Turtle race, their prices are in a down trend and the recent Bud buffs made us fear their inflation even more.

Without new boost NFTs at stake, most players avoid digging since it's seen as gambling. The only ones that keep digging daily are probably looking for extra XP with a chance to recover the costs with some good loots using shovels. No drills, just shovels, meaning only wood and stones are used by them.

When we look at TI as a reference, there are some things we should consider in order when releasing the new fishing feature:

1) Players will only spend their resources when there is a clear reward to go for 
2) Gold and Iron are only burned when an utility NFT is on stake 

With those facts in mind:

> how can we motivate players to use not only wood and stone, but also iron and gold on a daily basis?

Watching the Fishing lives and after seeing the Fishing Rod's cost, I had an idea that could increase iron and gold burns on a daily basis and could be applied to either Fishing and Treasure Island tools.

### [PROPOSAL]

Introduce different tier tools and the concept of [tool durability] to the game.

**TOOL TIERS:**

Regular Tools (the ones we have access now): Single use
Iron Tools: Can be used 5 times
Gold Tools: Can be used 20 times

To help creating a "floor price" for the entire chain, the recipes proposed consider the price the player is paid by the Blacksmith when delivering him wood, stones, iron and gold. Blacksmith rewards per piece around:

**Wood:** 0,0625
**Stone:** 0,23
**Iron:** 1
**Gold:** 4,5

To incentivize players to burn gold and iron on a daily basis, high tier tools will have a better performance compared to the regular ones, meaning it will be cheaper to use when gold and iron prices are down (compared to the price paid by the Blacksmith)

Since stone's current price seems to be aligned to the value paid by the NPC (0,23 SFL), the idea consists in substitute stones for iron and gold in the new recipes.

### [RECIPES]

<img width="943" alt="Screenshot 2023-10-16 at 02 47 12" src="https://github.com/sunflower-land/sunflower-land/assets/11824506/8d93cf21-eda1-4777-8c52-caeee8794b39">

### NEW ITEM: FURNACE

To unlock the new tools, the player would have to buy a FURNACE from Plaza's Blacksmith. We could make it quest related, so we can introduce the feature. Furnace's recipe would ask mostly for woods and stones and would definitely be required due current discounts.

### PROS

Implementing this system would add a new strategy layer when buying tools. In the short term, the big discount by using iron and gold tools would correct the entire chain of mineral resources, while keeping the wood demand stable.

Since iron and gold are harder to obtain, the time aspect would play a bigger role on the mineral chain and will help to organically create a new floor for the entire mineral chain close the the ones set by the game (Blacksmith Delivery Rewards).

Players will have an organic and constant incentive to different minerals everyday, based on market prices flutuation.
